CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- Decimal type support (#203).
```

Comment:
It would be great to add a section about supported types in the documentation.

Reply:
It's rather simple and you don't have to do anything: if you receive a response with `decimal` from Tarantool, it would be converted to a Python `decimal` and vice versa. You don't need to do any additional imports like in go-tarantool.

On the other hand, you are right about the lack of documentation and examples. We will solve this as a part of https://github.com/tarantool/tarantool-python/issues/67 ticket which is planned after adding a datetime type (work in progress).

Reply:
> if you receive a response with `decimal` from Tarantool, it would be converted to a Python `decimal` and vice versa

This is where it may not be obvious. For example, I would expect some internal type in the package.

Reply:
Type mapping and restrictions would be added in https://github.com/tarantool/tarantool-python/issues/67 solution